pyrokinetics/equilibrium.py

```diff
@@ -159,6 +162,203 @@ def read_geqdsk(self, psi_n_lcfs=1.0):
 
         self.R_major = InterpolatedUnivariateSpline(psi_n, R_major)
 
+    def read_transp_cdf(
+        self, time=-1, nr=None, nz=None, Rmin=None, Rmax=None, Zmin=None, Zmax=None
+    ):
+        """
+
+        Read in TRANSP netCDF and populates Equilibrium object
+
+        """
+
+        import netCDF4 as nc
+        import numpy as np
+        from scipy.interpolate import (
+            RBFInterpolator,
+            InterpolatedUnivariateSpline,
+            RectBivariateSpline,
+        )
+
+        data = nc.Dataset(self.eq_file)
```

Comment:
It's best to always open files using a `with` context manager, to ensure they get closed.